dbt_meshify/utilities/versioner.py

```diff
@@ -58,11 +62,11 @@ def get_latest_model_version(model_versions: NamedList) -> int:
         except ValueError:
             raise ValueError("Version not an integer, can't increment version.")
 
-    def generate_version(
-        self, model: ModelNode, prerelease: bool = False, defined_in: Optional[Path] = None
-    ) -> ChangeSet:
-        """Create a Change that adds a Version config to a Model."""
+    def add_version(self, model: ModelNode, defined_in: Optional[Path] = None):
```

Comment:
not a big deal but it looks like we lost the return typehint for this method!

Reply:
Ugh. Gotta add that back.